standupbot.js

```diff
@@ -89,13 +52,15 @@ app.get('/', function(req, res) {
   }
 
   if (req.cookies.lastID) {
-    getStatusForID(req.cookies.lastID, function(err, result) {
+    knex('statuses').where({stats: req.cookies.lastID}).then(function(result) {
       for (var i=0; i<result.length; i++) {
         var row = result[i],
             key = STATES[row.state];
         locals[key].push(row.status);
       }
       render(locals);
+    }).catch(function(err) {
+      // TODO handle this error
```

Comment:
and this is also an HTML endpoint, so returning json might not be the best idea :)